public/js/graph/react_graph.js

```diff
@@ -895,19 +932,73 @@ var EdgeGroup = React.createClass({
                      source={edgeJSON.source}
                      target={edgeJSON.target}
                      points={edgeJSON.points}
-                     svg={this.props.svg} />;
+                     svg={this.props.svg}
+                     edgeID={edgeJSON.id_}
+                     updateEdgeStatus={this.updateEdgeStatus} />;
     },
 
     render: function () {
+        // var missingEdges = [];
+        // var otherEdges = [];
+        var edges = this.props.edgesJSON;
+        var edgesCopy = $.extend([], edges);
+        var state = this.state;
+        edgesCopy.sort(function(a, b) {
+            var aID = a.id_;
+            var bID = b.id_;
+            var aMiss = false;
+            var bMiss = false;
+            if (aID in state) {
+                if (state.aID) {
```

Comment:
1. You should combine this check with the previous one with an AND
2. Same comment as before. You need to check whether `state.aID` is missing.